transforms/dec128.js

```diff
@@ -14,10 +14,20 @@ const opToName = {
 };
 
 const replaceWithDecimalExpression = (t, knownDecimalNodes) => (path) => {
```

Comment:
I realize we are calling this function in both dec128 and big, and there is no difference (I think I thought there might be eventually). I should fix it, but I am resisting doing so in this PR to keep the unrelated noise down.